src/random.cpp

```diff
@@ -11,7 +11,7 @@
 #include <compat.h> // for Windows API
 #include <wincrypt.h>
 #endif
-#include <logging.h>  // for LogPrint()
+#include <logging.h>  // for LogPrintf()
 #include <sync.h>     // for WAIT_LOCK
```

Comment:
note: Will update 5cc4f20c7064fee90a78a827a9e30b2934ce4ca7 to change `WAIT_LOCK` to `Mutex`.